zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/util/SqlSplitter.java

```diff
@@ -106,39 +106,30 @@ public SqlSplitter(String... additionalSingleCommentPrefixList) {
               && text.length() > (index + 1)) {
         if (isSingleLineComment(text.charAt(index), text.charAt(index + 1))) {
           singleLineComment = true;
-        } else if (text.charAt(index) == '/' && text.charAt(index + 1) == '*') {
+        } else if (text.charAt(index) == '/' && text.charAt(index + 1) == '*'
+                && text.length() > (index + 2) && text.charAt(index + 2) != '+') {
```

Comment:
I would move `text.length() > (index + 2)` before `text.charAt(index + 1) == '*'`

Reply:
fixed

Reply:
sorry, I meant that we can write it as:

```
text.charAt(index) == '/' && text.length() > (index + 2) 
  && text.charAt(index + 1) == '*' && text.charAt(index + 2) != '+'
```

so we have range check for `index+1` covered, just in case if `index == lenght-1`...

Reply:
I see, now it is fixed